### PR TITLE
Update netease and geolocation-cn

### PR DESCRIPTION
**Pull request by EpLiar**

### CODE REVIEW

File: data/geolocation-cn [hunk 1]
```diff
@@ -88,6 +88,7 @@ moji.com
 3322.org
 35.com
 360doc.com
+360jq.com
```
Comment (iochen) [inline data/geolocation-cn:91]: why not add it to `qihoo360`

File: data/geolocation-cn [hunk 2]
```diff
@@ -320,12 +321,14 @@ linkvans.com
 linuxidc.com
 liuxiaoer.com
 lu.com
+ludashi.com
```
Comment (iochen) [inline data/geolocation-cn:324]: why not add it to `qihoo360`

File: data/geolocation-cn (hunk 1 above)
Comment (EpLiar) [reply data/geolocation-cn:91]: http://www.360jq.com/about.htm

According to the business license, this domain does not belong to Qihoo 360

Comment (iochen) [reply data/geolocation-cn:91]: > http://www.360jq.com/about.htm
> 
> According to the business license, this domain does not belong to Qihoo 360

sorry, my mistake

File: data/geolocation-cn (hunk 2 above)
Comment (EpLiar) [reply data/geolocation-cn:324]: It's my mistake, I'm going to modify it now.

**Comment by iochen** [comment]
fix #310

**Pull request closed without merging**